python/servo/build_commands.py

```diff
@@ -441,7 +441,7 @@ def build(self, target=None, release=False, dev=False, jobs=None,
             # Build the name of the package containing all GStreamer dependencies
             # according to the build target.
             gst_lib = "gst-build-{}".format(self.config["android"]["lib"])
-            gst_lib_zip = "gstreamer-{}-1.14.3-20181105-103937.zip".format(self.config["android"]["lib"])
+            gst_lib_zip = "gstreamer-{}-1.14.3-20190131-153818.zip".format(self.config["android"]["lib"])
```

Comment:
Not sure if you did that, but you also should ensure that (in addition to the libraries themselves) the relevant plugins you're going to use are also included. For WebRTC that are going to be quite a few.

Reply:
Hmm, I only added gstreamer-sdp-1.0 and gstreamer-webrtc-1.0 to the list of plugins, as you suggested before. I'm not sure which plugins we need for WebRTC.

Reply:
You added those libraries to the list of libraries, nothing with plugins. The plugins are in a different variable.

I'm not sure what you all need, would have to check that too. At least webrtc, rtp, rtpmanager, vpx, opus and nice.

Reply:
You need the following plugins: `opus vpx nice webrtc dtls srtp rtpmanager`. `sctp` too if/when you use data channels. I would recommend adding a runtime check that queries the gstreamer registry for the plugins that are available, something like this: https://github.com/centricular/gstwebrtc-demos/blob/master/sendrecv/gst-rust/src/main.rs#L113

Reply:
Ok, thanks!

We already have `opus` and `vpx`.

IIUC adding `GSTREAMER_PLUGINS_NET` to https://github.com/servo/libgstreamer_android_gen/blob/536676168014b60d59ce8a0d1ef4003632d9c941/jni/Android.mk#L30 should give us `webrtc dtls srtp` and `rtpmanager`. 

I don't see `nice` and `sctp` listed in `share/gst-android/ndk-build/plugins.mk` for 1.14.3.

Reply:
`sctp` is not available in 1.14.x, but `nice` being missing is a bug. I'll fix it our build aggregator, but in the meantime you can add it to the `GSTREAMER_PLUGINS_NET` list in `plugins.mk` by hand for now. The plugin should already be packaged correctly.

Reply:
The fix for `nice` has been [merged into 1.14](https://gitlab.freedesktop.org/gstreamer/cerbero/merge_requests/96) and will be available in 1.14.5, which should be out in the next few weeks.

Reply:
Great. Thank you! :)